oeps/oep-0045-arch-ops-and-config.rst

```diff
@@ -0,0 +1,152 @@
+##########################################
+OEP-45: Configuring and Operating Open edX 
+##########################################
+
+.. list-table::
+
+   * - OEP
+     - :doc:`OEP-0045 </oeps/oep-0045>`
+   * - Title
+     - Configuring and Operating Open edX
+   * - Last Modified
+     - 2020-04-06
+   * - Authors
+     - Bill DeRusha
+   * - Arbiter
+     - ???
+   * - Status
+     - Draft
+   * - Type
+     - Architecture
+   * - Created
+     - 2020-04-06
+   * - Review Period
+     - ???
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Abstract
+========
+
+Wherever possible, The edX organization will provide and manage Docker images as the medium for packaging operating dependencies of Open edX applications rather than providing Ansible playbooks.
+
+The configuration for running applications in different environments will be done through a config file per unique [application, environment] combination.  How those files are generated and managed is the responsibility of each operator.
+
+All standard operations for running and managing any Open edX application will be documented.  How those procedures are executed is the responsibility of each operator.
+
+Wherever possible, operating concerns will live within the codebase they are concerned with.  Thus it should be expected that all applications will provide the following:
+
+* A Dockerfile from which the container image can be built.
```

Comment:
This looks great, what are the plans with the devstack?, currently the yaml file used in the devstack is defined in https://github.com/edx/configuration/tree/master/docker/build for each service. Do you have any plans to use other repository? Maybe that can be in the service repo itself

Reply:
This should be answered [later in the OEP](https://github.com/edx/open-edx-proposals/pull/143/files#diff-9d5e701bc9f15777166f8459ecffb088R107), let me know if I can provide more clarity there.